src/utils.js

```diff
@@ -38,6 +38,8 @@ export function expressMiddleware (monitor, name) {
           monitor.measure(k, d[0] * 1000 + d[1] / 1000000);
           monitor.count(k);
         }
+        monitor.measure(['all', stat], d[0] * 1000 + d[1] / 1000000);
```

Comment:
I think you want to either move this inside the loop and end up with 
`taskcluster-queue.api.all.client-error.5m.count` and `taskcluster-queue.api.all.all.5m.count` or replace `stat` in these new lines with `success` and only end up with `taskcluster-queue.api.all.client-error.5m.count`.

I might be misunderstanding though.